src/Api/Vault/Models/Request/CipherRequestModel.cs

```diff
@@ -164,6 +168,7 @@ private CipherLoginData ToCipherLoginData()
             PasswordRevisionDate = Login.PasswordRevisionDate,
             Totp = Login.Totp,
             AutofillOnPageLoad = Login.AutofillOnPageLoad,
+            Fido2Key = Login.Fido2Key == null ? null : Login.Fido2Key.ToCipherLoginFido2KeyData(),
```

Comment:
Would this be cleaner if we use the shorthand?

`Fido2Key = Login.Fido2Key?.ToCipherLoginFido2KeyData();`

Reply:
I addressed this with https://github.com/bitwarden/server/pull/3317